src/editor/CSSInlineEditor.js

```diff
@@ -177,12 +169,33 @@ define(function (require, exports, module) {
 
         // Always use the selection start for determining selector name. The pos
         // parameter is usually the selection end.
-        var selectorResult = _getSelectorName(hostEditor, sel.start);
-        if (selectorResult.selectorName === "") {
-            return selectorResult.reason || null;
+        selectorResult.result = _getSelectorName(hostEditor, sel.start);
+        if (selectorResult.result.selectorName === "") {
+            return selectorResult.result.reason || null;
+        }
+        return true;
```

Comment:
This function seems confused.  Sometimes you return `null`, sometimes you return `selectorResult.result.reason`, which is what, a String?  Sometimes you return a boolean.  You should be consistent.